src/app/components/MainCatSection.jsx

```diff
@@ -0,0 +1,45 @@
+// This is the MainCatSection component which is responsible for rendering the main categories section of the website. It uses the categoriesData.json file to get the data for the categories.
+// Path of the data json file: src\app\data\MainCatSection\categoriesData.json
+
+import React from "react";
+import Image from "next/image";
+import categoriesData from "../data/MainCatSection/categoriesData.json";
+
+const CategoryCard = ({ src, alt, title, description }) => (
+  <div className="p-8 cursor-pointer">
+    <Image src={src} alt={alt} width={274} height={103} />
```

Comment:
Does this give responsive behavior for images? You may have to use tailwind classes here
And always have the object fit property to 'cover' in these scenarios